src/navigation/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Search from "../features/search";
+import camera from "../images/camera.svg";
+import { Wrapper } from "./styled";
+
```

Comment:
Nie ma tutaj żadnych importów styli z styled.js, więc to nie może działać. Musisz zaimportować wszystkie komponenty ze styled.js